misc/tokens.pl

```diff
@@ -220,3 +220,4 @@ sub normalize_name {
 0 0 0 0 0 x-forwarded-for
 0 0 0 0 0 x-traffic
 0 0 0 0 0 cache-digest
+0 0 0 0 0 compress-hint
```

Comment:
I removed the `x-` prefix because of https://tools.ietf.org/html/rfc7231.html#section-8.3.1 do you think we should still keep it?